diff -up python-fastapi-0.121.2/pyproject.toml.omv~ python-fastapi-0.121.2/pyproject.toml
--- python-fastapi-0.121.2/pyproject.toml.omv~	2025-11-18 23:16:25.028671891 +0100
+++ python-fastapi-0.121.2/pyproject.toml	2025-11-18 23:16:33.444553036 +0100
@@ -49,7 +49,7 @@ classifiers = [
     "Topic :: Internet :: WWW/HTTP",
 ]
 dependencies = [
-    "starlette>=0.40.0,<0.50.0",
+    "starlette>=0.40.0,<0.60.0",
     "pydantic>=1.7.4,!=1.8,!=1.8.1,!=2.0.0,!=2.0.1,!=2.1.0,<3.0.0",
     "typing-extensions>=4.8.0",
     "annotated-doc>=0.0.2",
